src/pyrqlite/connections.py

```diff
@@ -68,8 +68,8 @@ def _fetch_response(self, method, uri, body=None, headers={}):
             uri = response.getheader('Location')
             location = urlparse(uri)
 
-            logging.debug("status: %s reason: '%s' location: '%s'",
-                          response.status, response.reason, uri)
+            logging.getLogger(__name__).debug("status: %s reason: '%s' location: '%s'",
```

Comment:
Ditto -- get this variable outside the scope of any one function, and use as needed.